ldk-server-protos/src/proto/events.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+import "types.proto";
+package events;
+
+// Represents different types of events, useful for routing and filtering.
+enum EventType {
+  UNKNOWN = 0;
+  PAYMENT_FORWARDED = 1;
+}
+
+// EventEnvelope wraps different event types in a single message to be used by EventPublisher.
+message EventEnvelope {
+  EventType event_type = 1;
```

Comment:
Why do we need `EventType` if we are using `oneof`?

Reply:
Do you mean they would decided whether or not whether to deserialize the field based on `event_type`?  Can't the protobuf serve that purpose?